### PR TITLE
CAT-1300 Enable vibration brick tests

### PR DESCRIPTION
**Pull request by avihric**
This is a reopening of #1250 
Implementation was done by @BerndBaumann 

@robertpainsi @obusher @thmq  can someone look over this and merge it afterwards please.

### CODE REVIEW

**Comment by avihric** [comment]
retest this please

**Comment by robertpainsi** [comment]
retest this please

File: catroidTest/src/org/catrobat/catroid/uitest/content/brick/VibrationBrickTest.java [hunk 1]
```diff
@@ -58,74 +72,65 @@ protected void tearDown() throws Exception {
 		SensorTestServerConnection.closeConnection();
 		setActivityInitialTouchMode(true);
 		super.tearDown();
```
Comment (robertpainsi) [inline catroidTest/src/org/catrobat/catroid/uitest/content/brick/VibrationBrickTest.java]: `java.lang.NullPointerException`
https://jenkins.catrob.at/job/CatroidDeviceTest/2580/testReport/junit/org.catrobat.catroid.uitest.content.brick/VibrationBrickTest/testVibrationBrick/

File: catroidTest/src/org/catrobat/catroid/uitest/content/brick/VibrationBrickTest.java [hunk 2]
```diff
@@ -23,28 +23,42 @@
...
+	private Project project;
 
 	public VibrationBrickTest() {
 		super(ScriptActivity.class);
 	}
 
-/*	@Override
+	@Override
 	protected void setUp() throws Exception {
 		createProject();
```
Comment (robertpainsi) [inline catroidTest/src/org/catrobat/catroid/uitest/content/brick/VibrationBrickTest.java]: ```super.setUp();``` should be the first call (if possible)

**Pull request closed without merging**